src/Compilers/CSharp/Portable/Symbols/FunctionPointerTypeSymbol.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Reflection;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal class FunctionPointerTypeSymbol : TypeSymbol
```

Comment:
doesn't implement a corresponding IFunctionPointerTypeSymbol?

Reply:
> doesn't implement a corresponding IFunctionPointerTypeSymbol?

No, I'm leaving the public API for a future follow-up PR.

Reply:
> doesn't implement a corresponding IFunctionPointerTypeSymbol?

In addition to the other response, internal C# symbols no longer implement public symbols interfaces directly.

---
In reply to: [358685039](https://github.com/dotnet/roslyn/pull/40431#discussion_r358685039) [](ancestors = 358685039)